kubetools_client/config.py

```diff
@@ -52,11 +52,35 @@ def load_kubetools_config(
         dev (bool): filter config items by dev mode
     '''
 
-    possible_files = (
+    possible_filenames = (
         'kubetools.yml',
         'kubetools.yaml',
     )
 
+    if directory:
+        possible_files = [
+            path.join(directory, filename)
```

Comment:
Hmm looks like this logic is now duplicated on L89?

Reply:
Well, this is now also in line 69 and 80.
Looks to me like a perfect situation for a good old recursive call

Reply:
This should be implemented in https://github.com/EDITD/kubetools-client/pull/13/commits/dd76be1b4f424bbcf1ad2d050d6fa7009967cd0e!